kernel/kernel-api/src/test/scala/io/delta/kernel/internal/clustering/ClusteringMetadataDomainSuite.scala

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (2025) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.internal.clustering
+
+import java.util.{Collections, Optional, UUID}
+
+import scala.collection.JavaConverters._
+
+import io.delta.kernel.expressions.Column
+import io.delta.kernel.internal.actions.{Format, Metadata}
+import io.delta.kernel.internal.util.{ColumnMapping, ColumnMappingSuiteBase, VectorUtils}
+import io.delta.kernel.types._
+
+import org.scalatest.funsuite.AnyFunSuite
+
+class ClusteringMetadataDomainSuite
+    extends AnyFunSuite
+    with ColumnMappingSuiteBase {
+
+  private def createMetadata(schema: StructType): Metadata = new Metadata(
+    UUID.randomUUID.toString,
+    Optional.empty(),
+    Optional.empty(),
+    new Format,
+    schema.toJson,
+    schema,
+    VectorUtils.buildArrayValue(Collections.emptyList(), StringType.STRING),
+    Optional.empty(),
+    VectorUtils.stringStringMapValue(Collections.emptyMap()))
+
+  private def convertToPhysicalColumn(
+      logicalColumns: List[Column],
+      schema: StructType): List[Column] = {
+    logicalColumns.map { column =>
+      ColumnMapping.getPhysicalColumnNameAndDataType(schema, column)._1
+    }
+  }
+
+  test("ClusteringDomainMetadata could be serialized") {
```

Comment:
let's use a more active & present tense, instead of a past subjunctive tense. 

Can you say `can be serialized` instead of `could` ? Thanks!